lib/spack/spack/test/container/test_docker.py

```diff
@@ -0,0 +1,74 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
```

Comment:
Can we remove `test` from these file names? It just gets in the way of being able to read what we're testing, and when we're in the `test/container` directory I don't think it's helping us any.

Reply:
Right, I forgot to change the name of these files when I imported the command into core Spack from a custom extension. Will make this change. 

To explain the origin of the file naming and why I forgot to adapt it: this is not a convention I came up with but it is instead the [commonly accepted naming](https://docs.pytest.org/en/latest/goodpractices.html#conventions-for-python-test-discovery) when using pytest in a Python project. Spack customizes the default in its `pytest.ini`:

https://github.com/spack/spack/blob/b7f04935630cbe9c94f8bdc64bbace595c80cfc8/lib/spack/spack/test/pytest.ini#L2-L5

Reply:
Done